src/Http/samples/MinimalSample/MinimalSample.csproj

```diff
@@ -5,6 +5,7 @@
     <Nullable>enable</Nullable>
     <EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <IsTrimmable>true</IsTrimmable>
```

Comment:
Is this intended?

Reply:
Yes, since we don't have the integration tests, I enabled trimming on the sample project as another "canary in the coal mine" within the repo should the diagnostic suppressor break in any way.